browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -1799,6 +1809,9 @@ void RewardsDOMHandler::GetMonthlyReport(const base::ListValue* args) {
 
 void RewardsDOMHandler::OnGetAllMonthlyReportIds(
     const std::vector<std::string>& ids) {
+  if (!IsJavascriptAllowed())
```

Comment:
see https://github.com/brave/brave-core/pull/8772/files#r630337636
This will DCHECK in `CallJavascriptFunction` because this method should never be called before `AllowJavascript` is called